### ISSUE TITLE
Build Issue

### ISSUE THREAD

**Issue by bsc08** [Azure/Azure-Media-Services-Explorer#79]
Can't build 4.0.1.0 as is - WindowsAzure.Storage library included is 7.2.1 and the build seems to require 8.1.4. 

Once I manually reference 8.1.4, telemetry won't load with the message:
"Could not load file or assembly 'Microsoft.WindowsAzure.Storage, Version=7.2.1.0......The located assembly's does not match the assembly reference.

FYI - referencing 7.2.1 in the intial project will compile, but will crash if you try and use the storage key.

In other words, the build needs to be tested some more.
